docs/UserGuide.md

```diff
@@ -53,9 +66,9 @@ Example:
     3. Toss the spaghetti in the sauce,
     4. Taste and season with salt and black pepper.
 
-###### [return to table of content](#table-of-content)
+ [<sub>return to table of content<sub>](#table-of-content)
 
-### Edit recipe
+### Edit an existing recipe: `edit`
 Edits a recipe by specifying its name, ingredients, and steps to prepare the dish.<br>
 Usage: `edit -n <name of recipe> -d <completion time of recipe> -ss <serving size of recipe> -i <ingredients of recipes separated by commas> -s <steps separated by commas> -t <tags associated with recipe> (optional)` <br>
```

Comment:
Actually for this part (and the command summary) do you wanna do the update? Since it's your feature HAAHHA. As discussed earlier, we should probably standardise index to number for UG.